### ISSUE TITLE
False positives with *Error types

### ISSUE THREAD

**Issue by VasekPurchart** [slevomat/coding-standard#52]
Newly introduced checking of `*Error` types in [1.1](https://github.com/slevomat/coding-standard/releases/tag/1.1) has false positives, when a class is named `*Error`, but is not part of PHP's internal errors.

What's worse is that even PHP has class(es?) which are named like this, but are not an "exception".
In our code base I ran into:

```php
$xml = @simplexml_load_string($result);
if (!($xml instanceof SimpleXMLElement) || libxml_get_last_error() instanceof LibXMLError) {
	// ...
}
````

which then reports error on `LibXMLError`, which is just a "normal" class.

This could have probably occurred with `*Exception` as well, although it is probably less likely.

I think, there should be a configurable list of types which are *not* "exceptions" - similar to the configurations of special exception names, which provide inverse functionality.

**Comment by ondrejmirtes** [comment]
Should be fixed with #57, please test it.

**Comment by VasekPurchart** [comment]
Now it works correctly, all gren, thanks :)
